src/core/external/collect/collect-container.ts

```diff
@@ -121,6 +123,9 @@ class CollectContainer extends Container {
     required: false,
   }) => {
     validateCollectElementInput(input, this.#context.logLevel);
+    if (input.type === ElementType.FILE_INPUT) {
+      validateCollectElementFileInput(input, options);
```

Comment:
there is already validations for FILE_INPUT inside validateCollectElementInput()  use the function there.